apps/tup-ui/src/components/auth/LoginComponent/LoginComponent.module.css

```diff
@@ -2,37 +2,49 @@
 @import url('@tacc/core-styles/dist/settings/space.css');
 @import url('@tacc/core-styles/dist/settings/font.css');
 
+@import url('@tacc/core-styles/dist/elements/form.cms.css');
+
+
+
+/* CONTAINER */
+
 .root {
-  width: 507px;
   display: flex;
   flex-direction: column;
   align-items: stretch;
-  padding: 47px 55px 36px 55px;
-  border: var(--global-border--normal);
-  border-color: var(--global-color-primary--dark);
-  font-family: var(--global-font-family--serif);
-  background-color: var(--global-color-primary--xx-light);
```

Comment:
ℹ️ Moved to [apps/tup-ui/src/pages/Login/Layout.module.css](https://github.com/TACC/tup-ui/pull/47/files#diff-59eac9815575f1ca6157e3bce16c037388750c813da00da77e17cb5e3507a7c5) and tweaked.